template/macro/2015/stories.html

```diff
@@ -71,7 +71,7 @@
               <td height="5"></td>
             </tr>
             <tr valign="top">
-              {% if more_thumbs and (index == 2 or index == 3) %}
+              {% if (small_first_img and index == 1) or (more_thumbs and (index == 2 or index == 3)) %}
```

Comment:
This is going to have to be made sane at some point. Presumably the trail logic should be to declare the number of trails that have thumbnails.